CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ target_include_directories(${PROJECT_NAME} PUBLIC
     ${PROJECT_SOURCE_DIR}/src
 )
 
+include_directories(${PROJECT_SOURCE_DIR})
+include_directories(${PROJECT_SOURCE_DIR}/src)
+include_directories(${PROJECT_SOURCE_DIR}/src/Core)
```

Comment:
Including src directory allows us to also include headers from it's subdirectories.